Presentation.Web/app/components/it-contract/it-contract-overview.controller.ts

```diff
@@ -618,6 +620,22 @@
                         sortable: false,
                         filterable: false
                     },
+                    {
+                        field: "LastChangedName", title: "Sidst redigeret: Bruger", width: 150,
+                        persistId: "lastchangedname", // DON'T YOU DARE RENAME!
+                        template: dataItem => dataItem && dataItem.LastChangedByUser ? `${dataItem.LastChangedByUser.Name} ${dataItem.LastChangedByUser.LastName}` : "",
+                        hidden: true,
+                        sortable: false,
+                        filterable: false
+                    },
+                    {
+                        field: "LastChanged", title: "Sidste redigeret: Dato", width: 150,
+                        persistId: "lastchanged", // DON'T YOU DARE RENAME!
+                        template: dataItem => dataItem?.LastChanged ? dataItem.LastChanged.toString() : "",
```

Comment:
make sure you align with other places where we render dates in the kendo grid. I recall some helper

Notice the difference:
**Contracts:**
![image](https://user-images.githubusercontent.com/13116889/171377438-f2014ed3-1a79-4bbf-882a-8d24d57a1c36.png)

vs
**It-systems:**
![image](https://user-images.githubusercontent.com/13116889/171377560-a4894e4b-1160-4222-9dea-48ca7dd868b1.png)